gcloud/datastore/connection.py

```diff
@@ -493,8 +508,8 @@ def delete_entities(self, dataset_id, key_pbs):
         :type key_pbs: list of :class:`gcloud.datastore.datastore_v1_pb2.Key`
         :param key_pbs: The keys to delete from the datastore.
 
-        :rtype: boolean
-        :returns: `True`
+        :rtype: :class:`bool`
```

Comment:
I don't think we need `:class:` for builtins like `str`, `int`, or `bool` -- we won't be cross-referencing them anywhere (the major reason for that role, AFAIK).